### PR TITLE
262 remove rulestext

### PR DESCRIPTION
**Pull request by patrickwieth**

### CODE REVIEW

File: src/views/Discord.vue [hunk 1]
```diff
@@ -129,8 +129,6 @@ import { CouncilStatus } from "decentralcardgame-cardchain-client-ts/DecentralCa
 import axios from "axios";
 import { useUrlSearchParams } from '@vueuse/core'
 import { createReusableTemplate } from '@vueuse/core'
-import queryString from 'query-string';
```
Comment (lxgr-linux) [inline src/views/Discord.vue:132]: thx

File: src/model/Card.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export class ChainCard {
       card.Health = parseInt(content[cardType].Health);
       card.Attack = parseInt(content[cardType].Attack);
       card.Delay = parseInt(content[cardType].Delay);
-      card.RulesTexts = content[cardType].RulesTexts;
+      card.RulesTexts = [];
```
Comment (lxgr-linux) [inline src/model/Card.ts:42]: Let's completely remove it from the card type.

**Review by lxgr-linux — CHANGES_REQUESTED** [review]

File: src/model/Card.ts (hunk 2 above)
Comment (patrickwieth) [reply src/model/Card.ts:42]: hmm I think it is being used by the CardCreator to put the rulesText together when you are assembling abilities?

File: src/components/elements/CardComponent.vue [hunk 3]
```diff
@@ -23385,6 +23385,108 @@ export default {
...
+          effectText.push(R.join(" ", interactionBlock))
+        })
+        return effectText
+      }
+
+      let rules = this.cardRules.definitions
+      let getRule = key => this.cardRules.definitions[decapital(key)]
+      let getInteractionText = key => R.split(" ", rules[decapital(key)].interactionText)
+
+      let rulesText = ""
```
Comment (lxgr-linux) [inline src/components/elements/CardComponent.vue:23442]:
```suggestion
      let rulesText = []
```

Please empty array not empty string.
this method is supposed to return an array, not a string

**Pull request merged**